### PR TITLE
[TW-1488] Doc Provide more granular RBAC for various actions in Postman

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Updates the API roles section with the new API Admin role
- Adds various notes for when the API Admin role is required to perform an action (publish, move, delete)

View the beta docs here: 
- [API roles](https://tw-1488-api-admin-role.learning.postman-beta.com/docs/collaborating-in-postman/roles-and-permissions/#api-roles)
- [Moving elements to team workspaces](https://tw-1488-api-admin-role.learning.postman-beta.com/docs/collaborating-in-postman/working-with-your-team/collaborating-in-team-workspaces/#moving-elements-to-team-workspaces)
- [Deleting an API](https://tw-1488-api-admin-role.learning.postman-beta.com/docs/designing-and-developing-your-api/creating-an-api/#deleting-an-api)
- [Collaborating on APIs](https://tw-1488-api-admin-role.learning.postman-beta.com/docs/designing-and-developing-your-api/managing-apis/#collaborating-on-apis)
- [Publishing an API to consumers](https://tw-1488-api-admin-role.learning.postman-beta.com/docs/designing-and-developing-your-api/versioning-an-api/api-versions/#publishing-an-api-to-consumers)

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/designing-and-developing-your-api/creating-an-api.md [hunk 1]
```diff
@@ -75,7 +75,9 @@ To change the name of an API, select **APIs** in the sidebar. Select the more ac
 
 ## Deleting an API
 
+> You must have the [API Admin role](/docs/collaborating-in-postman/roles-and-permissions/#api-roles) to delete an API.
```
Comment (jatin3893) [inline src/pages/docs/designing-and-developing-your-api/creating-an-api.md:78]: API Admin role or Workspace Admin either works

File: src/pages/docs/designing-and-developing-your-api/creating-an-api.md [hunk 2]
```diff
@@ -75,7 +75,9 @@ To change the name of an API, select **APIs** in the sidebar. Select the more ac
...
+
 1. Select **APIs** in the sidebar.
 1. Select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to an API and select **Delete**.
 
-> When you delete an API, any API definitions or collections contained in the API are deleted with it. Configured integrations are also deleted.
+When you delete an API, any API definitions or collections contained in the API are deleted with it. Configured integrations are also deleted.
```
Comment (jatin3893) [inline src/pages/docs/designing-and-developing-your-api/creating-an-api.md:83]: Not sure if we need to specify this -> but some more side effects of deleting an API:
1. Even published versions would get deleted
2. It will get removed from the private API network as well

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md [hunk 3]
```diff
@@ -34,7 +34,9 @@ You can also publish an API version programmatically with [the Postman CLI](/doc
 
 Publish an API version to share the current state of your API with consumers. You can select the elements (definition and collections) to include with your published API.
 
-> If your API is connected to a Git repository, publishing a version syncs the selected elements (definition and collections) to your workspace in the Postman cloud.
+If your API is connected to a Git repository, publishing a version syncs the selected elements (definition and collections) to your workspace in the Postman cloud.
+
+> You must have the [API Admin role](/docs/collaborating-in-postman/roles-and-permissions/#api-roles) to publish an API.
```
Comment (jatin3893) [inline src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md:39]: API Admin role or Workspace Admin role

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 4]
```diff
@@ -166,29 +166,34 @@ The following roles control access at a collection level:
 
 ### API roles
 
-You can [assign](/docs/collaborating-in-postman/requesting-access-to-elements/#managing-element-based-roles) two role types in Postman APIs: **Editor** and **Viewer**.
+You can [assign](/docs/collaborating-in-postman/requesting-access-to-elements/#managing-element-based-roles) three role types in Postman APIs: **Admin**, **Editor**, and **Viewer**.
 
-* **Editor** - Can edit APIs directly
-* **Viewer** - Can view and export APIs
+* **Admin** - Can publish, move, and delete APIs
+* **Editor** - Can edit APIs and API definitions
```
[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply src/pages/docs/collaborating-in-postman/roles-and-permissions.md:172]: Not sure if there was another reason, but it's an SEO differentiator because searching on "edit API role" will return all false positives.

File: src/pages/docs/collaborating-in-postman/working-with-your-team/collaborating-in-team-workspaces.md [hunk 5]
```diff
@@ -51,36 +51,29 @@ You can invite team members to join a [new](/docs/collaborating-in-postman/using
 
 ## Moving elements to team workspaces
 
-You can move collections, APIs, environments, and Flows to shared workspaces to collaborate with your team.
+You can move collections, APIs, environments, and Flows to shared private, team, or public workspaces to [collaborate with your team](https://www.postman.com/api-platform/api-collaboration/).
 
-> You must be an Editor on a collection, API, environment, or Flow, the Workspace Admin, or a [Super Admin](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to move an element to another workspace.
+> You must be an Editor on a collection, environment, or Flow, the Workspace Admin, or a Super Admin to move the element to another workspace. For APIs, you must be the API Admin, the Workspace Admin, or a Super Admin. Learn more about [roles and permissions](/docs/collaborating-in-postman/roles-and-permissions/).
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/working-with-your-team/collaborating-in-team-workspaces.md:56]:
```suggestion
> You must be the Workspace Admin, a Super Admin, or an Editor on a collection, environment, or Flow to move the element to another workspace. For APIs, you must be the API Admin, the Workspace Admin, or a Super Admin. Learn more about [roles and permissions](/docs/collaborating-in-postman/roles-and-permissions/).
```

Comment (jkonrath-postman) [reply src/pages/docs/collaborating-in-postman/working-with-your-team/collaborating-in-team-workspaces.md:56]: Minor nit - it was hard to parse so I flipped it around.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/designing-and-developing-your-api/creating-an-api.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/creating-an-api.md:83]: @jatin3893 Thanks for the additional information! I've changes this to read:

When you delete an API, the following items are deleted with it:

* API definitions or collections contained in the API
* Published API versions (including versions published to the Private API Network)
* The Private API Network listing for the API
* Integrations configured for the API

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/collaborating-in-postman/roles-and-permissions.md:172]: You're right, no real need. I changed it to just "Edit APIs" 😀

File: src/pages/docs/designing-and-developing-your-api/creating-an-api.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/creating-an-api.md:78]: Per Jatin's feedback I changes this to "You must have the API Admin or Workspace Admin role to delete an API." I think it's important to specify "API Admin" since there are several different types of Admin roles.

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md:39]: I changes this to:

> You must have the Admin role for an API or the Workspace Admin role to delete an API.

File: src/pages/docs/designing-and-developing-your-api/creating-an-api.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/creating-an-api.md:78]: I changes this to:

> You must have the Admin role for an API or the Workspace Admin role to delete an API.

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md:39]: I changed this to:

> You must have the Admin role for an API or the Workspace Admin role to publish an API.

**Review by shashankawasthi88 — APPROVED** [review]
Approved
